templates/base/head_navbar.tmpl

```diff
@@ -82,31 +82,33 @@
 				</div>
 				<span class="only-mobile tw-ml-2">{{ctx.Locale.Tr "active_stopwatch"}}</span>
 			</a>
-			<div class="active-stopwatch-popup item tippy-target tw-p-2">
-				<div class="tw-flex tw-items-center">
-					<a class="stopwatch-link tw-flex tw-items-center" href="{{.ActiveStopwatch.IssueLink}}">
-						{{svg "octicon-issue-opened" 16 "tw-mr-2"}}
+			<div class="active-stopwatch-popup tippy-target">
+				<div class="tw-flex tw-items-center tw-gap-2 tw-p-3">
+					{{svg "octicon-issue-opened" 16}}
+					<a class="stopwatch-link tw-flex muted" href="{{.ActiveStopwatch.IssueLink}}">
```

Comment:
Wait, the svg is no longer part of the link?
Why?

Reply:
I moved it because it allows me to use the parent flexbox to vertically center. If we really want to make the svg clickable, then I need to `display: inline-flex` the link I guess to allow vertical centering.